### PR TITLE
Update the page on how to delete db

### PR DESCRIPTION
**Pull request by NataliaIvakina**
Fix the following: 

- the config - `dbms.directories.dumps.root` → `server.directories.dumps.root`
- `neo4j-admin dump` command → `neo4j-admin database dump`
- `neo4j-admin load` command → `neo4j-admin database load`

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1777.surge.sh

Updated pages:
- [Delete databases](https://neo4j-docs-operations-1777.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)

File: modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc [hunk 1]
```diff
@@ -57,28 +57,28 @@ SHOW DATABASES YIELD name
 | "system"            |
 +---------------------+
 ----
-======
+
 
 [[delete-databases-existing]]
-=== Use `IF EXISTS` when deleting databases
+=== Delete a database using `IF EXISTS`
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:63]:
```suggestion
=== Delete a database with `IF EXISTS`
```

File: modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc [hunk 2]
```diff
@@ -57,28 +57,28 @@ SHOW DATABASES YIELD name
...
 
-.Query
 [source, cypher]
 ----
 DROP DATABASE customers IF EXISTS
 ----
 
 [[manage-databases-dump]]
-=== Use `DUMP DATA` or `DESTROY DATA` when deleting databases
+=== Delete a database using `DUMP DATA` or `DESTROY DATA`
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:76]:
```suggestion
=== Delete a database with `DUMP DATA` or `DESTROY DATA`
```

File: modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc [hunk 3]
```diff
@@ -88,23 +88,22 @@ DROP ALIAS `motion pictures` FOR DATABASE;
...
 
-.Query
+
 [source, cypher]
 ----
 DROP DATABASE movies DUMP DATA
 ----
 
+[[delete-existing-db-with-dump]]
+=== Delete a database combining `IF EXISTS` and  `DUMP DATA`/ `DESTROY DATA`
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:98]:
```suggestion
=== Delete a database with `IF EXISTS` and  `DUMP DATA`/ `DESTROY DATA`
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1777.surge.sh

Updated pages:
- [Delete databases](https://neo4j-docs-operations-1777.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1777.surge.sh

Updated pages:
- [Delete databases](https://neo4j-docs-operations-1777.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)

File: modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc [hunk 4]
```diff
@@ -57,54 +58,49 @@ SHOW DATABASES YIELD name
...
 
-.Query
 [source, cypher]
 ----
 DROP DATABASE movies DUMP DATA
 ----
 
+These dumps are equivalent to those produced by xref:backup-restore/offline-backup.adoc[`neo4j-admin database dump`] and can be similarly restored using the xref:backup-restore/restore-dump.adoc[`neo4j-admin database load`] command.
+
+In Neo4j, dumps can be stored in the directory specified by the xref:configuration/configuration-settings.adoc#config_server.directories.dumps.root[`server.directories.dumps.root`] setting (by default, the path is xref:configuration/file-locations.adoc#data[`<neo4j-home>/data/dumps`]).
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc:90]:
```suggestion
In Neo4j, dumps can be stored in the directory specified by the xref:configuration/configuration-settings.adoc#config_server.directories.dumps.root[`server.directories.dumps.root`] setting (by default, the path for storing dumps is xref:configuration/file-locations.adoc#data[`<neo4j-home>/data/dumps`]).
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1777.surge.sh

Updated pages:
- [Delete databases](https://neo4j-docs-operations-1777.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)

**Pull request merged**